contribs/osm/src/main/java/org/matsim/contrib/osm/networkReader/LinkProperties.java

```diff
@@ -139,7 +147,7 @@ public static double getLaneCapacity(double linkLength, LinkProperties propertie
      * Calculate free speed of a link based on heuristic if it is an urban link.
      */
 	public static double calculateSpeedIfSpeedTag(double maxSpeed) {
-        double urbanSpeedFactor = maxSpeed < 51 / 3.6 ? 0.5 : 1.0; // assume for links with max speed lower than 51km/h to be in urban areas. Reduce speed to reflect traffic lights and suc
+        double urbanSpeedFactor = maxSpeed < 51 / 3.6 ? DEFAULT_FREESPEED_FACTOR : 1.0;
```

Comment:
if the urbanSpeed factor is supposed to be applied in ```afterLinkCreated``` then it shouldn't be applied here.